fdbserver/IKeyValueStore.h

```diff
@@ -173,7 +173,7 @@ inline IKeyValueStore* openKVStore(KeyValueStoreType storeType,
                                    bool checkChecksums = false,
                                    bool checkIntegrity = false,
                                    bool openRemotely = false) {
-	if (openRemotely) {
+	if (openRemotely && !storeType == KeyValueStoreType::SSD_ROCKSDB_V1) {
```

Comment:
@liquid-helium Is this required? Chaoguang Lin has pushed a PR to disable remote store for rocksdb.

Reply:
Reverted.